activities/Clock.activity/index.html

```diff
@@ -17,7 +17,16 @@
 <script>if (window.module) module = window.module;</script>
 <script data-main="js/loader" src="lib/require.js"></script>
 </head>
-
+<script>
+  function change() {
+    var abc = document.getElementById("seconds-button-before")
+    if (abc) {
+        abc.id = "seconds-button-after"
```

Comment:
Not a good idea to change the id. You could use two different classes and switch from one to another.